backend/src/utils/__init__.py

```diff
@@ -1,12 +1,13 @@
 from .config import Config
-from .llm import call_model, call_model_with_tools, get_response_three_prompts
+from .llm import call_model, call_model_with_tools, get_response_three_prompts, MistralClient
```

Comment:
Do we actually use `MistralClient` outside of the `/utils` directory?

Reply:
we use it in the tests